### ISSUE TITLE
how to do the line break using jspdf, with data taken from the database?

### ISSUE THREAD

**Issue by jpods1093** [simonbengtsson/jsPDF-AutoTable#499]
![image](https://user-images.githubusercontent.com/43278850/58489326-6c385800-8141-11e9-8c30-6389f9a95a2b.png)
my problem is with a column overlaying another column and I do not know how to play the rest of the text to the bottom line as a text break, below this my code for export with jspdf.

> 
}
        else if(defaults.type == 'pdf'){
          rowPositionHeader = 0;
          var doc = new jsPDF({orientation: 'l', unit: 'pt', format: 'a4',hotfixes: []});
          doc.setFontSize(7);

          $(el).find('thead').find('tr').each(function(index,data) {
            index2 = rowCalc = index+1;
            if (rowCalc % 35 == 0){
              doc.addPage();
              page++;
              startRowPosition=startRowPosition;
            }
            rowPositionHeader = rowPosition=(startRowPosition + (rowCalc * 10)) - ((page -1) * 280) -20;
           //calculation for the layout of the lines in pdf
            $(this).filter(':visible').find('th').each(function(index,data) {
              if ($(this).css('display') != 'none'){
                if(defaults.ignoreColumn.indexOf(index) == -1){
                  var colPosition = startColPosition+ (index * 130);
                  console.log("rowPosition: ", rowPosition);
                  doc.text(colPosition,rowPosition, parseString($(this)));
                }
              }
            });
          });

          // Row Vs Column
          var rowCalc = 0; startRowPosition = rowPositionHeader;

          $(el).find('tbody').find('tr').each(function(index,data) {
            rowCalc += 1;

            if (rowCalc % 50 == 0){
              doc.addPage();
              startRowPosition=10;
              rowPositionHeader = 0;
              rowPosition = 0;
              rowCalc = 5;  

        }
my code for export in pdf using jspdf autotable , how to make a breake line in this case the img , i dont now to make this

**Comment by jpods1093** [comment]
Simon you have idea what this problem ?

**Comment by simonbengtsson** [comment]
It's hard to understand if it is an issue or just a question. Can you make your ticket easier to understand? There is some steps you can follow in the issue template.

**Comment by jpods1093** [comment]
> 
> 
> It's hard to understand if it is an issue or just a question. Can you make your ticket easier to understand? There is some steps you can follow in the issue template.

i'm change the issue , Let me know if you're clearer to understand.

**Comment by simonbengtsson** [comment]
Still hard to follow. Can you recreate the issue with 10 code lines or so?

**Comment by jpods1093** [comment]
> 
> 
> Still hard to follow. Can you recreate the issue with 10 code lines or so?

let me try to explain my problem, well I'm doing a system where I already have a database fed and my system does searches in this database and has the options to export both for pdf and excel, when a column has a word too large or several things one column disrupts the other and do not know how to do the line break to solve this problem. I do not know if it was clear from the issue
